src/Account.js

```diff
@@ -14,14 +21,28 @@ export default class Account {
     this._mpeContract = mpeContract;
   }
 
+  /**
+   * Returns the token balance available.
+   * @returns {Promise.<BigNumber>}
+   */
   async balance() {
+    logger.info('Fetching account balance', { tags: ['Account'] });
```

Comment:
Should be debug ?

Reply:
Yeah, I think we can mark a bunch of other `logs` as debug. I will follow the following rule to differentiate between `debug` and `info` logs

`info` -> Any critical operation which is likely to have side effects
`debug` -> All other logs

Does the above splitting look good?

Reply:
Sounds good